src/components/protocol_handler/test/protocol_handler_tm_test.cc

```diff
@@ -732,7 +743,7 @@ TEST_F(ProtocolHandlerImplTest,
   AddSecurityManager();
 
   EXPECT_CALL(session_observer_mock, KeyFromPair(connection_id2, session_id2))
```

Comment:
@ValeriiMalkov good place for ON_CALL, please take in mind

Reply:
@LuxoftAKutsan The `session_observer_mock` declared as a `StrictMock` object. Maybe we leave it like that ?